pkg/constants/constants.go

```diff
@@ -100,4 +100,8 @@ const (
 
 	NotAvailableLabel = "Not available"
 	BackendCmd        = "avalanche-cli-backend"
+
+	AvalancheGoCompatibilityURL  = "https://raw.githubusercontent.com/ava-labs/avalanchego/master/version/compatibility.json"
```

Comment:
also add RPC here?

Reply:
not sure what you mean?

Reply:
both SubnetEVMRPCCompatibilityURL and SpacesVMRPCCompatibilityURL are explicit regarding compatibility refers to RPC version compatibility, but AvalancheGoCompatibilityURL not